setup.cfg

```diff
@@ -19,17 +19,6 @@ ignore =
 per-file-ignores =
     core/mocks/generated/*:F4
 
-[isort]
-multi_line_output = 3
-include_trailing_comma = True
-force_grid_wrap = 0
-combine_as_imports = True
-line_length = 88
-not_skip=__init__.py
-known_standard_library = micropython,ubinascii,ustruct,uctypes,utime,utimeq,trezorio,trezorui,trezorutils,trezorconfig,trezorcrypto
-known_third_party = curve25519,ecdsa,hypothesis,pytest,mnemonic,shamir-mnemonic,trezor,storage,protobuf
-known_first_party = trezorlib,apps
```

Comment:
it looks like the new isort is good enough to figure out the "3rd party" stuff on its own, but we would like to keep both `known_standard_library` and `known_first_party`, see the other comments